uk.co.agilemde.agileuml.yaml

```diff
@@ -0,0 +1,80 @@
+id: uk.co.agilemde.agileuml
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk21
+command: agileuml
+finish-args:
+  - --socket=x11
+  - --filesystem=host
```

Comment:
Why do you need filesystem host?

Reply:
I'm not sure, actually. I will provisionally use "filesystem=home" for now. Sorry for not responding to this earlier.